### PR TITLE
[Refactor][GUI] Set static texts in .ui files + add missing tr()

### PR DESCRIPTION
**Pull request by random-zebra**
Set initial / static texts for widgets and dialogs in the .ui files instead of in the .cpp files, with `notr` where needed. Also add `tr()` where missing (ref: #1600).
Finally clean up styling (missing spaces, branches on newlines).

### CODE REVIEW

File: src/qt/pivx/coldstakingwidget.cpp [hunk 1]
```diff
@@ -46,7 +46,7 @@ class CSDelegationHolder : public FurListRow<QWidget*>
         QString address = index.data(Qt::DisplayRole).toString();
         QString label = index.sibling(index.row(), ColdStakingModel::OWNER_ADDRESS_LABEL).data(Qt::DisplayRole).toString();
         if (label.isEmpty()) {
-            label = "Address with no label";
+            label = PIVXGUI::tr("Address with no label");
```
Comment (furszy) [inline src/qt/pivx/coldstakingwidget.cpp:49]: No need to be PIVXGUI, can be `QObject::tr()`.

File: src/qt/pivx/forms/addresseswidget.ui [hunk 2]
```diff
@@ -386,12 +390,16 @@
              <item>
               <widget class="QLabel" name="labelAddress">
                <property name="text">
-                <string>Enter address</string>
+                <string>Enter PIVX address</string>
```
Comment (furszy) [inline src/qt/pivx/forms/addresseswidget.ui:393]: kinda redundant to be saying "Enter PIVX address" from the PIVX wallet. We can leave it as "Enter address".

File: src/qt/pivx/forms/dashboardwidget.ui [hunk 3]
```diff
@@ -454,7 +454,7 @@
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
-            <string/>
+            <string>How to get PIV or zPIV</string>
```
Comment (furszy) [inline src/qt/pivx/forms/dashboardwidget.ui:457]: "How to get PIV" only. No more zPIV.

File: src/qt/pivx/mnrow.cpp [hunk 4]
```diff
@@ -21,12 +21,8 @@ void MNRow::updateView(QString address, QString label, QString status, bool wasC
 {
     ui->labelName->setText(label);
     ui->labelAddress->setText(address);
+    if (!wasCollateralAccepted) status = tr("Collateral tx not found");
     ui->labelDate->setText("Status: " + status);
```
Comment (furszy) [inline src/qt/pivx/mnrow.cpp:25]: Missing status translation --> `tr("Status %1").arg(status);`

**Comment by random-zebra** [comment]
All comments addressed

**Pull request merged**